Makefile

```diff
@@ -149,6 +149,15 @@ gen-docs: gen-docs-metadata ## Generate docs for atlascli commands
 	@echo "==> Generating docs"
 	go run -ldflags "$(LINKER_FLAGS)" ./tools/cmd/docs
 
+.PHONY: gen-purls
+gen-purls: # Generate purls on linux os
+	@echo "==> Generating purls"
+	GOOS=linux GOARCH=amd64 go build -trimpath -mod=readonly -o bin/atlas-linux ./cmd/atlas
+	go version -m ./bin/atlas-linux | \
+		awk '$$1 == "dep" || $$1 == "=>" { print "pkg:golang/" $$2 "@" $$3 }' | \
+		LC_ALL=C sort > build/package/purls.txt
+	rm bin/atlas-linux
```

Comment:
you don't need to rm, the folder is ignored and it doesn't hurt to keep them around